src/pages/Home/Home/GetArticles.ts

```diff
@@ -0,0 +1,36 @@
+export default async function getArticles() {
+  const articles = sessionStorage.getItem("articles");
+
+  if (articles !== null) {
+    const articlesObject: any[] = JSON.parse(articles!);
+
+  return articlesObject.map((key) => ({
+    title: key.title,
+    description: key.description,
+    url: key.url,
+  }));
+  };
+
+  const responseFirstPage = await fetch(
+    "https://api.marketaux.com/v1/news/all?symbols=TSLA%2CAMZN%2CMSFT&filter_entities=true&limit=3&page=1&language=en,he&api_token=yRViEWPsrvESjsph62PVTHAp1hL4AgoXcv6NUBVF"
+  );
+  const articlesFirstPage = await responseFirstPage.json();
+
+  const responseSecondPage = await fetch(
```

Comment:
The only thing that changed between responseFirstPage and responseSecondPage is the page number . 

Its very specific and will be hard to change . I want you to make this more generic (lets say i want 12 articles what would you need to do to make it happen with the current function ?)